dev/ci/ci-basic-overlay.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+# This is the basic overlay set for repositories in the CI.
+
+# Maybe we should just use Ruby to have real objects...
```

Comment:
That doesn't offset the pain that bash is, moreover when ruby is standard in Travis.